common/tasks/fifo_scheduler_test.go

```diff
@@ -210,7 +210,7 @@ func (s *fifoSchedulerSuite) TestStartStopWorkers() {
 	processor.stopWorkers(numWorkers / 2)
 	s.Len(processor.workerShutdownCh, numWorkers/2)
 
-	processor.stopWorkers(len(processor.workerShutdownCh))
+	processor.stopWorkers(len(processor.workerShutdownCh) + 1)
```

Comment:
Is there a reason we have a list of shutdown channels instead of a single shutdown channel we can close?

Reply:
Idea is dynamically control the # of workers, so don't want to always shutdown all the workers. 

One single shutdown channel can work, if it's buffered with size > # of workers we ever want to have and we push X tokens into it to shutdown X workers and close it on host shutdown. 

I guess I felt separate channels is easier to implemented when writing this since I don't need to differentiate reducing worker and host shutdown.